packages/twilio-run/src/templating/defaultConfig.ts

```diff
@@ -0,0 +1,68 @@
+import camelCase from 'lodash.camelcase';
+import path from 'path';
+import { Options } from 'yargs';
+import { ALL_FLAGS } from '../flags';
+import { fileExists, writeFile } from '../utils/fs';
+import { getDebugFunction } from '../utils/logger';
+
+const debug = getDebugFunction('twilio-run:templating:defaultConfig');
+
+function renderDefault(config: Options): string {
+  if (config.type === 'boolean') {
+    if (typeof config.default === 'boolean') {
+      return config.default.toString();
+    }
+    return 'false';
+  } else if (config.type === 'string') {
+    if (typeof config.default === 'string') {
+      return `"${config.default}"`;
+    }
+    return 'null';
+  }
+
+  return 'null';
+}
+
+function templateFlagAsConfig([flag, config]: [string, Options]) {
+  return `\t// "${camelCase(flag)}": ${renderDefault(config)} \t/* ${
+    config.describe
+  } */,`;
+}
+
+function templateDefaultConfigFile() {
+  const lines = Object.entries(ALL_FLAGS)
+    .sort((a, b) => a[0].localeCompare(b[0]))
+    .map(templateFlagAsConfig)
+    .join('\n');
+  return [
+    '{',
+    `\t"commands": {},`,
+    `\t"environments": {},`,
+    `\t"projects": {},`,
+    lines,
+    '}',
+  ].join('\n');
```

Comment:
Any reason to not use `JSON.stringify` here?

Reply:
Because I'm adding comments since it's JSON5

Reply:
Ah, in `templateFlagAsConfig`, got you. Shame JSON5 doesn't make it easier to actually work with the comments it permits.